website/content/docs/agent-and-proxy/agent/apiproxy.mdx

```diff
@@ -1,12 +1,11 @@
 ---
 layout: docs
-page_title: Vault Agent API Proxy
+page_title: Use Vault Agent as an API proxy
```

Comment:
We might not want to improve this page's SEO, as this is deprecated and will be removed in a later release

Reply:
That's probably true of a lot of pages if I'm honest, I'd rather just move through all the docs systematically than try to pick and choose. But I'll definitely add a deprecation notice to the top of the page.